# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1333728754 25200
#      Fri Apr 06 09:12:34 2012 -0700
# Node ID ad2ff36996b87e4b985c4881bb6ead976d65e572
# Parent  e6aa044253eb72dcb52b407f45ae63f5e547f7e9
NEWS: Add note about redundant serial comma feature in parser

* NEWS: Add note about redundant serial comma feature in parser

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,11 +1,21 @@
 Summary of important user-visible changes for version 3.8:
 ---------------------------------------------------------
 
+ ** Redundant terminal comma accepted by parser
+
+    A redundant terminal comma is now accepted in matrix
+    definitions which allows writing code such as 
+
+    [a,...
+     b,...
+     c,...
+    ] = deal (1,2,3)
+
  ** Warning IDs renamed:
 
     Octave:array-as-scalar => Octave:array-to-scalar
     Octave:array-as-vector => Octave:array-to-vector
 
  ** Other new functions added in 3.8.0:
 
       colorcube   splinefit
